src/main/java/robosky/uplands/item/ItemRegistry.java

```diff
@@ -0,0 +1,66 @@
+package robosky.uplands.item;
+
+import net.minecraft.entity.effect.StatusEffects;
+import net.minecraft.item.AliasedBlockItem;
+import net.minecraft.item.FoodComponent;
+import net.minecraft.item.Item;
+import net.minecraft.util.registry.Registry;
+import robosky.uplands.UplandsMod;
+import robosky.uplands.block.BlockRegistry;
+
+public class ItemRegistry {
+
+    public static Item AEGISALT_CRYSTAL;
+
+    public static Item AEGISALT_CHARM;
+    public static Item SKYROOT_CHARM;
+
+    public static Item ZEPHYR_ONION_ITEM;
+    public static Item ROASTED_ZEPHYR_ONION_ITEM;
+
+    public static Item WATER_CHESTNUT_ITEM;
+    public static Item WATER_CHESTNUT_SEEDS_ITEM;
+
+    public static Item SKYROOT_FRUIT;
+
```

Comment:
We generally initialize these immediately so the fields are able to be final.

Reply:
Okay, made it final.